packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -228,8 +225,9 @@ export const getStyles = memoizeFunction(
     const placeholderStyles: IStyle = {
       color: semanticColors.inputPlaceholderText
     };
-    const hoverPlaceholderStyles: IStyle = {
-      color: palette.neutralPrimary
+    const placeholderStylesHovered: IStyle = {
+      // color: palette.neutralPrimary //o

```

Comment:
Remove comment 😄

Reply:
Oh jeeze I thought I got all of those, thanks!